src/OmniSharp.Host/Services/OmniSharpEnvironment.cs

```diff
@@ -38,6 +40,20 @@ public class OmniSharpEnvironment : IOmniSharpEnvironment
             TraceType = traceType;
             TransportType = transportType;
             OtherArgs = otherArgs;
+
+            // On Windows: %APPDATA%\OmniSharp\omnisharp.json
```

Comment:
Wasn't there a huge fight about that when `dotnet` was still `dnx`?  And they moved things to `APPDATA` because that was the "windows way"?  Some people wanted `.dotnet` and others wanted it elsewhere?

Personally I like using `USERPROFILE` myself, but I don't to make things any more confusing than they have to be.